File: Components/Util/ValidationHelpers.vb
Public Class ValidationHelpers
    Public Shared Function CommonValidator(ByVal key As String) As String
        Select Case key
            Case "Integer"
                Return "[-|+]?\b\d+\b"
            Case "PositiveInteger"
                Return "\b\d+\b"
            Case "Color", "Colour"
                Return "^([a-fA-F0-9]{6}|[a-fA-F0-9]{3})$"
        End Select
        Return key
    End Function
End Class

File: Components/Util/ValidationHelpers.cs





public class ValidationHelpers
{
	public static string CommonValidator(string key)
	{
		switch (key)
		{
			case "Integer":
				return "[-|+]?\\b\\d+\\b";
			case "PositiveInteger":
				return "\\b\\d+\\b";
			case "Color":
			case "Colour":
				return "^([a-fA-F0-9]{6}|[a-fA-F0-9]{3})$";
		}
		return key;
	}
}

